source/cloud-security/azure/activity-services/prerequisites/credentials.rst

```diff
@@ -35,14 +35,8 @@ Getting access credentials for Storage
     :align: center
     :width: 100%
 
-
-Authentication options
-----------------------
-
-There are two different ways to set up the Azure authentication:
-
 Using an authentication file
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----------------------------
 
 It is possible to store the credentials in a file for authentication as long as the file content follows the `field = value` format explained below.
```

Comment:
Now that the `Authentication options` title was removed, this subtitle lacks some context. I would suggest doing something like

```
Authentication
---------------

To authenticate, store the credentials in a file whose content follows the `field = value` format explained below.
```